examples/input/gamepad_input.rs

```diff
@@ -1,12 +1,13 @@
 use bevy::prelude::*;
+use bevy_app::startup_stage::POST_STARTUP;
 use bevy_input::gamepad::{Gamepad, GamepadButton, GamepadEvent, GamepadEventType};
 use bevy_utils::HashSet;
 
 fn main() {
     App::build()
         .add_default_plugins()
         .init_resource::<GamepadLobby>()
-        .add_startup_system(connection_system.system())
+        .add_startup_system_to_stage(POST_STARTUP, connection_system.system())
```

Comment:
I'm assuming you're adding this duplicate system here in case gamepad_system runs before connection system?

Could we just have a single `add_system_to_stage(PRE_UPDATE, connection_system.system())` instead?

Reply:
`.add_system_to_stage(stage::PRE_UPDATE, connection_system.system())` works fine (tested it too)